From 0a75cb118b615ff57a408bccbcc31e2be0095a83 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Sun, 6 Jan 2019 19:28:54 +0200
Subject: [PATCH 08/12] Revert "sepolicy: Allow wcnss_service to set
 wlan.driver properties"

* Will be replaced by upstream commits

This reverts commit 22832f10abbd8f7715ffbe5540a3f97a04f1dc95.

Change-Id: I29e16bdbc93051575074fa21d1f20fd1e34e5134
---
 common/property.te       | 2 --
 common/property_contexts | 3 ---
 common/wcnss_service.te  | 2 --
 3 files changed, 7 deletions(-)

diff --git a/common/property.te b/common/property.te
index b1f037a..7881756 100644
--- a/common/property.te
+++ b/common/property.te
@@ -136,5 +136,3 @@ type mmi_prop, property_type;
 
 #imsrcsservice
 type ctl_vendor_imsrcsservice_prop, property_type;
-
-type wcnss_prop, property_type;
diff --git a/common/property_contexts b/common/property_contexts
index acd71c4..3625b0f 100644
--- a/common/property_contexts
+++ b/common/property_contexts
@@ -130,6 +130,3 @@ persist.vendor.crash.detect        u:object_r:crash_prop:s0
 persist.vendor.mmi.        u:object_r:mmi_prop:s0
 
 ctl.vendor.imsrcsservice   u:object_r:ctl_vendor_imsrcsservice_prop:s0
-
-wlan.driver.ath            u:object_r:wcnss_prop:s0
-wlan.driver.config         u:object_r:wcnss_prop:s0
diff --git a/common/wcnss_service.te b/common/wcnss_service.te
index a8f957a..e07d6df 100644
--- a/common/wcnss_service.te
+++ b/common/wcnss_service.te
@@ -78,5 +78,3 @@ hal_client_domain(wcnss_service, hal_perf)
 
 # read persist.vendor.wigig.npt.enable
 get_prop(wcnss_service, wigig_prop);
-
-set_prop(wcnss_service, wcnss_prop)
-- 
2.17.1

